src/scripts/send-magic-link.ts

```
@@ -0,0 +1,31 @@
+import { createClient } from '@supabase/supabase-js'
+
+// Replace with your Supabase URL and anon key
+const supabaseUrl = process.env.VITE_SUPABASE_URL!
+const supabaseAnonKey = process.env.VITE_SUPABASE_ANON_KEY!
+
+const supabase = createClient(supabaseUrl, supabaseAnonKey)
+
+async function sendMagicLink(email: string) {
+  try {
+    const { data, error } = await supabase.auth.signInWithOtp({
```

ESLint

[error] 'data' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)